modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -49,6 +49,10 @@ controller:
     %{~ if !allow_snippet_annotations ~}
     annotation-value-word-blocklist: load_module,lua_package,_by_lua,location,root,proxy_pass,serviceaccount,{,},',\
     %{~ endif ~}
+    deny-default-annotation-blocker: ${deny-default-annotation-blocker}
+    %{~ if allow_snippet_annotations && deny-default-annotation-blocker ~}
```

Comment:
Shouldn't we delete row 49 etc as well?  `%{~ if !allow_snippet_annotations ~}`
Else we will have the same value 2 times or am I reading wrong?